page-changelog.php

```diff
@@ -2,6 +2,167 @@
+	</li></ul>
+<li>Chunked Files: Optimize checking if all chunks are there <a href="https://github.com/owncloud/core/issues/22601">#22601</a></li>
+<li>Remove browser autocomplete in new file menu <a href="https://github.com/owncloud/core/issues/22784">#22784</a></li>
+<li>Integrity check stumbles over lost+found special directory <a href="https://github.com/owncloud/core/issues/23341">#23341</a></li>
+<li>Code integrity failure when modifying .user.ini <a href="https://github.com/owncloud/core/issues/23144">#23144</a></li>
+<li>Admin page down after update to 9.0 <a href="https://github.com/owncloud/core/issues/22960">#22960</a></li>
+<li>Remove browser autocomplete in new file menu <a href="https://github.com/owncloud/core/issues/22784">#22784</a></li>
+<li>Redirect loop on session timeout when exploring trashbin <a href="https://github.com/owncloud/core/issues/21140">#21140</a></li>
+<li>Fix tooltip for versions mtime <a href="https://github.com/owncloud/core/issues/22924">#22924</a></li>
+<li>The download archive should be named after the name of the current folder <a href="https://github.com/owncloud/core/issues/22836">#22836</a></li>
+<li>Undefined index: newVersionString <a href="https://github.com/owncloud/core/issues/22917">#22917</a></li>
+<li>Rename and move permissions are set when a file is updatable <a href="https://github.com/owncloud/core/issues/22841">#22841</a></li>
+<li>Files Sidebar shows no entries <a href="https://github.com/owncloud/core/issues/22980">#22980</a></li>
+<li>Class 'XMLReader' not found in Reader.php (after upgrade from 8.2.2) <a href="https://github.com/owncloud/core/issues/23003">#23003</a></li>
+<li>Fix call to disk_free_space when a file is provided <a href="https://github.com/owncloud/core/issues/22912">#22912</a></li>
+<li>Missing document icons in filter views <a href="https://github.com/owncloud/core/issues/23044">#23044</a></li>
+<li>Remove disabled autocorrect for new file names <a href="https://github.com/owncloud/core/issues/23024">#23024</a></li>
+<li>Getting a 404 when calling url/owncloud; fine when calling url/owncloud/index.php <a href="https://github.com/owncloud/core/issues/22970">#22970</a></li>
+<li>Call to a member function getRequest() on null <a href="https://github.com/owncloud/core/issues/23031">#23031</a></li>
+<li>Windows Office files via SMB mount through WebDAV is prompting for authentication <a href="https://github.com/owncloud/core/issues/22596">#22596</a></li>
+<li>php fatal error during upgrade from 8.2.2.2 to 9.0.0.1 <a href="https://github.com/owncloud/core/issues/23020">#23020</a></li>
+<li>500 on PROPFIND with "oc:owner-display-name" <a href="https://github.com/owncloud/core/issues/23116">#23116</a></li>
+<li>Explicitly check for port <a href="https://github.com/owncloud/core/issues/23156">#23156</a></li>
+<li>PHP 5.4 Warning Wording Needs Updating <a href="https://github.com/owncloud/core/issues/1170">#1170</a></li>
```

Comment:
@jospoortvliet Do you know where this issue / link is coming from? I havn't found any references to that wording and https://github.com/owncloud/core/issues/1170 is a quite old and unrelated issue.